theano/gpuarray/tests/check_dnn_doc.py

```diff
@@ -0,0 +1,284 @@
+"""
```

Comment:
I'll rename it. These helper functions are intended to be used to check if specifications listed into cuDNN documentation about algorithms are correct (ie. if doc says FWD/FFT algo should fail in certain conditions, check that it indeed fail), that's why I had named this file "check dnn doc".